src/connector/src/parser/plain_parser.rs

```diff
@@ -66,7 +67,7 @@ impl PlainParser {
         };
 
         let transaction_meta_builder = Some(AccessBuilderImpl::DebeziumJson(
-            DebeziumJsonAccessBuilder::new()?,
+            DebeziumJsonAccessBuilder::new(TimestamptzHandling::GuessNumberUnit)?,
```

Comment:
Given it is system metadata I believe `[1973-03-03 09:46:40, 5138-11-16 09:46:40)` is enough. Additionally, the structure of the metadata may be static rather than dynamic, so maybe we can `derive(serde::Deserialize)` rather than accessing a json `Value`. But this can be a separate issue.